fetch: #namespace

    # Fetch Task Realated parameters
    n_actions: 6 # elbow_flex_joint+/-,shoulder_lift_joint+/-,shoulder_pan_joint+/-
    n_observations: 3 # XYZ of the TCP and the distance from GOAL

    max_iterations: 20

    init_pos: # This has to be validated in the fetch_moveit_test.py in fetch_openai_ros_example or something that tests this pos is possible
      bellows_joint: 0.0
      elbow_flex_joint: 0.0
      forearm_roll_joint: 0.0
      head_pan_joint: 0.0
      head_tilt_joint: 0.0
      l_gripper_finger_joint: 0.0
      r_gripper_finger_joint: 0.0
      shoulder_lift_joint: 0.0
      shoulder_pan_joint: 0.0
      torso_lift_joint: 0.0
      upperarm_roll_joint: 0.0
      wrist_flex_joint: 0.0
      wrist_roll_joint: 0.0

    goal_pos: # This has to be validated in the fetch_moveit_test.py in fetch_openai_ros_example or something that tests this pos is possible
      elbow_flex_joint: -1.8
      shoulder_lift_joint: 0.7
      shoulder_pan_joint: 1.32


    position_delta: 0.1 # Increments of Decrements in the positions each action step

    reached_goal_reward: 100
